src/qml/FeatureListForm.qml

```diff
@@ -172,7 +172,10 @@ Rectangle {
 
     delegate: Rectangle {
       id: itemBackground
-      anchors { left: parent.left; right: parent.right }
+      anchors {
+        left: parent ? parent.left : undefined
```

Comment:
@nirvn Is there a better way to prevent the errors withing delegates trying to access their parent? When the delegates get rerendered, there is time when the parent is `null`.  I have tried https://stackoverflow.com/questions/61605859/qml-delegatemodel-package-makes-delegate-have-null-parent-transiently , but didn't manage to make it work with anchors. I guess @signedav  might be interested too, since we have discussed workarounds with him and concluded this seems the simplest.